test/NonSiloTests/SerializationTests/StreamTypeSerializationTests.cs

```diff
@@ -0,0 +1,217 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Serialization.Formatters.Binary;
+using Orleans.Providers.Streams.AzureQueue;
+using Orleans.Providers.Streams.Common;
+using Orleans.Runtime.Host.Providers.Streams.AzureQueue;
+using Orleans.Serialization;
+using Orleans.ServiceBus.Providers;
+using OrleansServiceBus.Providers.Streams.EventHub;
+using Xunit;
+
+namespace UnitTests.Serialization
```

Comment:
In the azure queue stream provider, we deserialized the container using
DerializationManager.DeserializeFromByteArray<AzureQueueBatchContainer>(bytes);

My suspicion is that this will work even if we've serialized a AzureQueueBatchContainer2, but can we add a test to verify?